docs/concepts/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Basic Concepts Overview 
+headline: Spine Basic Concepts
+bodyclass: docs
+layout: docs
+type: markdown
+sidenav: doc-side-concepts-nav.html
+---
+<h2 class="top">Spine Basic Concepts</h2>
+
+<div id="toc" class="toc hide-block"></div>
+This section introduces key concepts of Spine framework and its parts, helps you get a deeper understanding of how it works and provides some usage notes when creating Spine-based cloud application. 
+
+We assume that you are familiar with the basics of Domain-Driven Design. 
+
+Below you can find a typical Spine Event Engine application architecture; its building blocks are described below.
+
```

Comment:
Do we have a picture here?

If the picture is not ready, please have a placeholder text.